docs/tests/integration/test_fim/test_registry/test_registry_basic_usage/test_basic_usage_registry_duplicated_entries.md

```diff
@@ -0,0 +1,35 @@
+# Test duplicated regitry entries
+
+This test will check that for two monitored registries with the same name value but with different capitalisation only triggers one added event when the registry is created.
```

Comment:
Typo in word capitalization. 
```suggestion
This test will check that for two monitored registries with the same name value but with different capitalization only triggers one added event when the registry is created.
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1247/commits/1c9bfd087282e2435cc7f5f770ca5786abe95752)